### ISSUE TITLE
Need to create org and then create user in tox test for Users.

### ISSUE THREAD

**Issue by thequbit** [mediapublic/mediapublic#89]
To really test the user end point, we should be adding the user with a valid organization_id.  I do not know how to create an organization first, and then reference it's ID to create a user.  @ryansb could you shed some light on that?

**Comment by ryansb** [comment]
Likely the best strategy would be to seed the database using fixed UUIDs.

**Comment by thequbit** [comment]
Gotcha.  Perhaps a flag in the ini file ( development ) and then do that in `scripts/initializedb.py` if it's set?  I think that method makes sense to me.

**Comment by ryansb** [comment]
No, it'd be as part of the gabbi setup in `server/mediapublic/tests/api.py` since it's _only_ for API tests.
